packages/calypso-products/src/constants/features.ts

```diff
@@ -19,6 +19,8 @@ import {
 export const FEATURE_WP_SUBDOMAIN = 'wordpress-subdomain';
 export const FEATURE_BLOG_DOMAIN = 'blog-domain';
 export const FEATURE_CUSTOM_DOMAIN = 'custom-domain';
+export const FEATURE_CUSTOM_DOT_BLOG_DOMAIN = 'custom-dot-blog-domain';
```

Comment:
Why are we adding this constant/feature with this change? It's related to the product feature this supports, but should it be added with the rest of the .blog domain product work?

Reply:
sorry I'm going to remove this constant.